### PR TITLE
Allow installing extensions by dist-tag

### PR DESCRIPTION
**Pull request by Nokel81**
Signed-off-by: Sebastian Malton <sebastian@malton.name>

### CODE REVIEW

File: src/renderer/components/+extensions/extensions.tsx [hunk 1]
```diff
@@ -490,18 +494,12 @@ export class Extensions extends React.Component<Props> {
...
-            when(() => !ExtensionInstallationStateStore.anyInstalling)
-              .then(() => this.installPath = "");
-          }
-        } finally {
-          prevSize = curSize;
+      reaction(() => ExtensionLoader.getInstance().userExtensions.size, (curSize, prevSize) => {
+        if (curSize > prevSize) {
+          disposeOnUnmount(this, [
+            when(() => !ExtensionInstallationStateStore.anyInstalling, () => this.installPath = ""),
+          ]);
```
Comment (Nokel81) [inline src/renderer/components/+extensions/extensions.tsx:502]: This is unrelated to the fix, just some cleanup of the same file.

**Review by jim-docker — APPROVED** [review]
